.github/workflows/build-assets.yml

```diff
@@ -335,10 +327,10 @@ jobs:
           build-args: |
             PKG_FILE=${{ steps.ln-container.outputs.k8s_pkg_file }}
           tags: |
-            ${{ secrets.JFROG_REGISTRY }}/${{ steps.vars.outputs.docker_tag_internal }}
-            ${{ secrets.JFROG_REGISTRY }}/${{ steps.vars.outputs.docker_tag_internal_base }}
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.internal_tag }}
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.candidate_base_branch }}
```

Comment:
dummy push to nowhere so we don't break builds on repositories that don't access jfrog?

Changes look good.

Reply:
Correct - there could be an argument to make the "push" boolean be a function of the JFROG_REGISTRY secret existing.